app/components/task-selection/component.js

```diff
@@ -22,6 +22,33 @@ export default class TaskSelectionComponent extends Component {
   @service store;
   @service tracking;
 
+  /**
+   * The manually selected customer
+   *
+   * @property {Customer} _customer
+   * @private
+   */
+  @localCopy("args.initial.customer")
+  _customer = this.args.initial.customer;
```

Comment:
```suggestion
  @localCopy("args.initial.customer")
  _customer;
```

You don't need to assign the argument manually, [as you can see in the docs](https://github.com/tracked-tools/tracked-toolbox?tab=readme-ov-file#localcopy).